docusaurus/docs/cloud/projects/collaboration.md

```diff
@@ -0,0 +1,57 @@
+
+To invite someone to collaborate on a project:
+
+1. Go to the *Overview* tab of your project dashboard.
+2. Click on the **Share** button in the top right hand corner.
+3. In the *Share [project name]* dialog, type the email address of the person to invite in the textbox. A dropdown indicating "Invite [email address]" should appear.
+4. Click on the dropdown: the email address should be displayed in a purple box right below the textbox.
+5. (optional) Repeat steps 3 and 4 to invite more people. Email addresses can only entered one by one but invites can be sent to several email addresses at the same time.
+6. Click on the **Send** button.
+
+People invited to collaborate on a project will be sent an email indicating how to join the project <!-- Question: What's on the email? Steps to follow? URL to click on? -->. Once a project is shared, <!-- Questions: How do we call the things that represent the people? Dots? Circles? Do they show profile pictures or only initials? --> will be displayed in the project dashboard next to the **Share** button, to see how many users collaborate on that project and who they are. <!-- Questions: What happens when we hover on those dots? Also, are they only displayed for the project owner or to everyone? -->.
+
+<!-- SCREENSHOT OF DASHBOARD WITH DOTS AND SHARE BUTTON -->
+
+## Managing invited users
+
+From the *Share [project name]* dialog accessible by clicking on the **Share** button of a project dashboard, projects owners can view the full list of people who have been invited to collaborate on the project. From there, it is possible to see the current status of each invited user and to manage those users.
+
+<!-- SCREENSHOT OF DIALOG WITH PEOPLE AND STATUS -->
+
+Invited users whose full name is displayed and dot is blue are users who did activate their account following the invitation email <!-- Not sure that's the correct wording -->. If however there are users in the list whose email address is displayed and dot is grey, it means they haven't activated their accounts and can't access the project dashboard yet. In that case, a status should be indicated right next to the email address to explain the issue:
+
+- Pending: the invitation email has been sent but the person hasn't acted on it yet.
+- Expired: the email has been sent ... time ago <!-- Question: After how long does the invitation expire? --> and the invitation expired.
+
+For both Pending and Expired statuses, it is possible to send another invitation email by clicking on the **Manage** button, then **Resend invite**. 
+
+### Revoking invitations
+
+To revoke an invited user's access to the project dashboard:
+
+1. Click on the **Share** button in the project dashboard.
+2. In the list of *People with access*, find the user whose access to revoke and click on the **Manage** button.
+3. Click on the **Revoke** button. <!-- Question: Is is done after clicking on the button or is there another confirmation dialog or something? -->
```

Comment:
<img width="381" alt="Screenshot 2023-08-01 at 10 54 39" src="https://github.com/strapi/documentation/assets/131254618/a7c1c300-0855-46d0-83d7-86b16e491227">

After clicking Revoke they see this confirmation modal